pkg/utils/utils.go

```diff
@@ -24,11 +24,13 @@ import (
 	"strings"
 
 	"github.com/golang/glog"
+	"github.com/jaypipes/ghw"
 )
 
 var (
 	sysBusPci = "/sys/bus/pci/devices"
```

Comment:
since we're adding sysDir, we can change sysBusPci to something like:
sysBusPci  = sysDir + '/bus/pci/devices'